kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/orchestration/ContainerConfiguration.java

```diff
@@ -201,6 +203,17 @@ public int getImageDownloadTimeoutSeconds() {
         return this.imageConfig.getimageDownloadTimeoutSeconds();
     }
 
+    /**
+     * return the container's network configuration as a
+     * {@link ContainerNetworkConfiguration}.
+     * 
+     * @return
+     * @since 2.7
```

Comment:
The `@since` tag should be set to the version of the `org.eclipse.kura.api` bundle that introduces the new API, so I think it should be 2.4 here. Probably the tag should be changed also in other places (all of the new APIs that have been introduced since Kura 5.1.0 release).